### ISSUE TITLE
Request: add system vimrc to vim

### ISSUE THREAD

**Issue by bartlibert** [ScoopInstaller/Main#749]
On my work laptop, $HOME is set to a slow network disk, so I use the $VIM/vimrc file to override this.
However, since vimrc is not persistent (while _vimrc is), I have to recreate this after every update.
Would it be possible to add "vimrc" (no _ in front) to the persist file for vim and vim-nightly?

**Comment by p1gsy** [comment]
customize your pwsh PROFILE, Make it easier.

vim $PROFILE

**Comment by bartlibert** [comment]
If you use gvim, no powershell is involved, so irrelevant...

Op za 25 jan. 2020 13:05 schreef p1gsy <notifications@github.com>:

**Comment by p1gsy** [comment]
maybe you don't need to do anything for this,
take a look: $VIM\_gvimrc
this file is defined as persistent.

**Comment by bartlibert** [comment]
_vimrc is loaded too late for $HOME modification to work, it needs to be vimrc, trust me, I checked.

**Comment by p1gsy** [comment]
so, you are put _vimrc in the $HOME?

**Comment by bartlibert** [comment]
Some things in vim, for example rtp, are set based on the HOME variable. As they are set before _vimrc is loaded, I have to modify vimrc instead, otherwise the rtp is wrong. I tried modifying the rtp afterwards, but some plugins are not happy with this. 
I hope I have stated my case enough now, so just let me know: is it allowed to be done (I'll make a pull request if necessary), or should I just fork this and be done with it?
